tests/baselines/reference/user/npm.log

```diff
@@ -326,6 +326,8 @@ node_modules/npm/lib/help.js(196,26): error TS2339: Property 'commands' does not
 node_modules/npm/lib/help.js(197,22): error TS2339: Property 'deref' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/lib/help.js(199,14): error TS2345: Argument of type 'any[]' is not assignable to parameter of type 'never'.
 node_modules/npm/lib/help.js(199,22): error TS2339: Property 'commands' does not exist on type 'typeof EventEmitter'.
+node_modules/npm/lib/help.js(206,33): error TS2339: Property 'length' does not exist on type 'never'.
+node_modules/npm/lib/help.js(207,14): error TS2339: Property 'split' does not exist on type 'never'.
```

Comment:
No more implicit any when indexing never (repros in ts too):

```ts
function f(l: never, x: any) {
  var x = l[0] // should be implicit any error here
  x.length  // and no error here
}
```

With noImplicitAny on, you should get a noImplicitAny error, and instead you get an error on the next line that 'length' does not exist on type 'never'.

Filed #26493

Reply:
Intended effect of https://github.com/Microsoft/TypeScript/pull/26340